examples/Transports/HTTP/http_basic.py

```diff
@@ -1,5 +1,5 @@
 """Tests HTTP requests between a server and a client."""
-import json
+import json, time
```

Comment:
For consistency, please push `import time` onto a new line. Wondering if this import is used. Probably was there for local testing?